scripts/drone/steps/lib.star

```diff
@@ -853,12 +853,13 @@ def playwright_e2e_tests_step():
             "PROV_DIR": "/grafana/scripts/grafana-server/tmp/conf/provisioning",
         },
         "name": "playwright-plugin-e2e",
-        "image": images["playwright"],
+        "image": images["ubuntu"],
         "depends_on": [
             "grafana-server",
         ],
         "commands": [
             "sleep 10s",  # it seems sometimes that grafana-server is not actually ready when the step starts, so waiting for a few seconds before running the tests
```

Comment:
maybe use 

```"npx wait-on@7.2.0 http://$HOST:$PORT",``` instead of sleep?

Similar to https://github.com/grafana/grafana/blob/d56a9ff5f7e7792729de436747ab3c0956bd529b/scripts/drone/steps/lib.star#L681C9-L681C48

Reply:
Nice! I thought we already did [`wait-for-grafana`](https://github.com/grafana/grafana/blob/chore/playwright-deps/scripts/grafana-server/wait-for-grafana/#L11) in the grafana-server step, but apparently that only happens in the cypress step.